.circleci/config.yml

```diff
@@ -106,22 +107,34 @@ jobs:
             done
             kubectl --context production version
             kubectl --context production cluster-info
+            kubectl --context production get pods --all-namespaces
       - run:
-          name: fix RBAC
+          name: log into ECR
           command: |
-            # make default account cluster-admin
-            kubectl create clusterrolebinding add-on-cluster-admin --clusterrole cluster-admin --serviceaccount=kube-system:default
+            aws_account_id=$(aws sts get-caller-identity --query "Account" --output text)
+            aws ecr get-login-password --region "$AWS_DEFAULT_REGION" | \
+              docker login --username AWS \
+                --password-stdin ${aws_account_id}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com
       - run:
-          name: dump cluster-info
+          name: setup regcred for staging minikube/k8s
           command: |
-            kubectl cluster-info
-            kubectl get pods --all-namespaces
+            kubectl --context staging create secret generic regcred \
+              --from-file=.dockerconfigjson=${HOME}/.docker/config.json \
```

Comment:
I was a little confused as to where this file comes from, but it seems from the [doc](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials) that it's placed there by `docker login` in the previous step. 👍🏽

Reply:
yeah i think so.